packages/components/src/components/drawer/drawer.jsx

```diff
@@ -15,87 +16,69 @@ const IconDrawer = ({ className }) => (
     </svg>
 );
 
-class Drawer extends React.PureComponent {
-    constructor(props) {
-        super(props);
-        this.state = { is_open: props.is_open };
-    }
+const Drawer = ({
+    anchor = 'left',
+    children,
+    className,
+    is_open,
+    toggleDrawer,
+    contentClassName,
+    footer,
+    header,
+    width = 250,
+    zIndex = 4,
+}) => {
+    const [has_is_open, setIsOpen] = useStateCallback(is_open);
```

Comment:
Can we maybe call it `has_open` if that's what it refers to? `has_is_open` looks misleading to me. Also, let's match the setter method name with the variable, e.i. `setHasOpen`.

Reply:
changed, all good, thanks 🙏